src/WormholeRouter.sol

```diff
@@ -135,9 +142,9 @@ contract WormholeRouter {
      */
     function settle(bytes32 targetDomain, uint256 batchedDaiToFlush) external {
         bytes32 sourceDomain = domains[msg.sender];
-        require(sourceDomain != bytes32(0), "WormholeRouter/sender-not-gateway");
+        if (sourceDomain == bytes32(0)) revert SenderNotDomain(msg.sender, sourceDomain);
```

Comment:
This can just report the `msg.sender`. `sourceDomain` will always be `bytes(0)` in this case.

Reply:
done in 0172295